cli/Cargo.toml

```diff
@@ -18,7 +18,8 @@ clap = { version = "4.0.9", features = ["derive"], optional = true }
 log = "0.4.17"
 thiserror = "1.0.31"
 futures = "0.3.21"
-pyro = { package = "pyroscope", version = "0.3.1", optional = true }
+pyro = { package = "pyroscope", version = "0.5.3", optional = true }
+pyroscope_pprofrs = { version = "0.2", optional = true }
```

Comment:
How are these changes related to the libp2p upgrade?

Reply:
`continuous-integration/gitlab-check-dependent-polkadot` job was failing in https://github.com/paritytech/substrate/pull/13587 because cargo was not able to find a suitable version of `wasm-bindgen-futures` (if I remember correctly) <- `reqwest` crate <- `pyroscope` crate.